20 UI update the UI of cloud upload buttons

Anishali2 opened the pull request:
![image](https://github.com/user-attachments/assets/96f3bb25-4bc1-4847-a0d6-387d567e413f)


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **Style**
  - Updated the shareable link button with a smoother hover effect for better visual feedback.
  - Streamlined cloud provider buttons, offering a clearer layout and explicit "Google Drive" and "DropBox" labels.

- **Refactor**
  - Simplified the cloud save section to provide a more intuitive and consistent user experience while retaining visible loading indicators.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

evereq commented on apps/portal/components/pagesComponents/_singleImageScreen/SingleImagePageManageAreaTemplate.tsx:
```diff
@@ -634,60 +629,48 @@ const SingleImagePageManageAreaTemplate: React.FC<Props> = ({
                             path="/images/google-drive-logo.svg"
                             size="25px"
                           />
-                          <div className={styles.providerText}>
```
@Anishali2 when you remove such blocks of code, always describe why you removing please! Where it now display those "Open image" and "Google drive" etc!?

Anishali2 replied:
Sir the onMouse Leave and ref is used for animations and I removed that now 
Also I've added the 'Open Image' and also 'Google Drive' is written there

evereq commented on apps/portal/components/pagesComponents/_singleImageScreen/SingleImagePageManageAreaTemplate.tsx:
```diff
@@ -710,15 +693,11 @@ const SingleImagePageManageAreaTemplate: React.FC<Props> = ({
                         >
+                          DropBox
                           <AppSvg path="/images/dropbox-logo.svg" size="25px" />
-                          <div className={styles.providerText}>
```
@Anishali2 when you remove such blocks of code, always describe why you removing please! Where it now display those "Open image" and "Google drive" etc!?

Anishali2 replied:
Sir the onMouse Leave and ref is used for animations and I removed that now 
Also I've added the 'Open Image' and also 'Google Drive' is written there 

I've removed "Open Image" text while testing and forget to add again.

evereq commented:
@Anishali2 this PR is big problem - it removes some features without explaining why / how it will work now etc. We need screenshots / videos showing how it was before and how it will work after PR get merged, especially when you clearly removing code / logic in PR!

evereq commented:
@Anishali2 
1) so why you removed animations, I don't get it?
2) bad that you removed something during testing and forgot to add it back. To fix those issues, just review your PR after you created it, EACH line, before you submit for review by others
3) please explain EACH line you removed in this PR, I still don't see any sense in most of changes in this PR, sorry

evereq commented on apps/portal/components/pagesComponents/_singleImageScreen/SingleImagePageManageAreaTemplate.tsx:
```diff
@@ -634,60 +632,52 @@ const SingleImagePageManageAreaTemplate: React.FC<Props> = ({
-                        className={styles.appButton}
-                        twPadding="tw-px-0 tw-py-3"
-                        twRounded="tw-rounded-none"
+                      <button
```
@Anishali2 is there any specific reason why we replaced AppButton with "button" in this line, but keep it AppButton in other places? Why we replaced it / touched it?

Anishali2 commented:
Before.

[Apex_1739309643812.webm](https://github.com/user-attachments/assets/7c9ae35e-435e-44ec-ac04-51f888dc2888)

After.

[Apex_1739309834555.webm](https://github.com/user-attachments/assets/fec72661-2da2-467b-a23a-ec44f94fabfc)

I've removed animations because animations looks not soo smooth + ui feels so much much boring and I've already mentioned it on google docs.

I removed AppButton component because this button is specifically used for this type of buttons + animations and I don't want that level of complexity to change that button which breaks the whole button structure so I decided to make a new one also those button looks too much old and those are totally not re-usable type of buttons maybe we should consider switching our theme to shad-cn in future in order to follow fully component base hierarchy 

+ As you seen in the video above you see there is hover effect which is not appear before.

Anishali2 commented:
Sorry Sir that's totally my mistake that I've removed the AppButton. I rechecked the whole project and I realized that this button is used everywhere so added that again don't worry this won't happen again.

MeNoKVN commented:
I looked through this repo earlier today and cloned & setup the firebase project. But i am not sure if i would update ui I feel like there are more issues with the actual navigation. For example clicking on these folders causes a full page rerender. I actually fixed the navigate to folder by changing some of on press events and improved some loading state but honestly i gave up. I found the structure a bit confusing. And also I was getting errors in almost every page.

Anishali2 commented:
> I looked through this repo earlier today and cloned & setup the firebase project. But i am not sure if i would update ui I feel like there are more issues with the actual navigation. For example clicking on these folders causes a full page rerender. I actually fixed the navigate to folder by changing some of on press events and improved some loading state but honestly i gave up. I found the structure a bit confusing. And also I was getting errors in almost every page.

Can you show us that errors which appears on every page because I don't see that maybe it's because of you didn't configure the project successfully.

As far as navigation and re-rendering stuff I'll fix that later but it's not as important right now until @evereq suggest to fix.

Some of the major issues are already mentioned on https://github.com/ever-co/ever-rec/issues if some issues are beyond of that let us know. :)

evereq commented:
@MeNoKVN, thank you for looking into the project and for your feedback. We built it a few years ago (it was a private project) and only recently switched to the latest React / Next 14, so there might be many issues related to the massive updates we did before making it open-source. We are currently adding more resources to this project to speed up fixing. If you have some minutes, we would highly appreciate it if you could report issues you find in GitHub https://github.com/ever-co/ever-rec/issues, that would help us a lot to stabilize the platform!

evereq commented:
@Anishali2 for navigation / re-rerendering we will fix those as part of migration to Next v15, but you can create ticket for that today too and mention all of those issues you can see

evereq commented:
@Anishali2 I watched video in https://github.com/ever-co/ever-rec/pull/34#issuecomment-2652137012, the reason we had those animations is because we will put much more icons into that initial button, not only 2 providers, but will be say 5. How you would solve it with your 2 buttons? Instead of removing that "feature", you could look into why those animations not work well / try to fix it, keeping exactly same user experience - one large button with few icons, hover on one of them show button with provider name, etc. Sorry, but I am going to close this PR without merge. If you can fix animations and make existing functionality to work well, please create a separate PR.
